### ISSUE TITLE
foter - accessibility -  hover state for 'need to get in touch' fails colour contrast

### ISSUE THREAD

**Issue by Dan-KeduTesting** [Buckinghamshire-Digital-Service/hr-service-desk-alpha#157]
see screenshot 

![fotter_-_accessibility_-__hover_state_for_'need_to_get_in_touch'_fails_colour_contrast](https://user-images.githubusercontent.com/2722522/65815321-f07d1380-e1e5-11e9-84be-f3e68a4247d6.png)

**Comment by enginehouse** [comment]
I think this is a caching issue - I can't reproduce this
